.ai-review.example.yml

```
@@ -0,0 +1,153 @@
+    - "..\\"
+    # Serialization
+    - "pickle.load"
+    - "pickle.loads"
+    - "yaml.load"
+    - "marshal.load"
+    # Weak crypto
+    - "md5"
+    - "sha1"
+    - "DES"
+    - "RC4"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)